utils/dbConnect.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import mongoose from "mongoose";
+
+
+const connection = {};
+
+async function dbConnect() {
+  if (connection.isConnected) {
+    return;
+  }
+
+  try {
+    if (!process.env.MONGODB_URI) {
+      throw new Error('MONGODB_URI environment variable is not set.');
+    }
+
+    await mongoose.connect(process.env.MONGODB_URI);
+    console.log("Mongodb connected")

```

Comment:
Nice job tucking a console in there that confirms the connection! The more (needed) console logs we have, the better!